reports/Oregon.md

```diff
@@ -434,7 +434,7 @@ tags: arrest, grab, journalist, threaten
 
 id: or-portland-7
 
-geolocation: 
+geolocation: 45.515557, -122.676821 // Chapman Square caught on video
```

Comment:
Same here - can you make notes in the description, rather than comments on the same line